components/AreaChartRenderer/index.ts

```diff
@@ -0,0 +1,7 @@
+import {
+  AreaChartRenderer,
+  AreaChartProps,
+} from 'components/AreaChartRenderer/AreaChartRenderer';
+
+export default AreaChartRenderer;
```

Comment:
I consider we can avoid default export with following
https://nextjs.org/docs/advanced-features/dynamic-import#with-named-exports